staging/src/k8s.io/client-go/tools/events/eventseries_test.go

```diff
@@ -168,17 +168,17 @@ func TestEventSeriesf(t *testing.T) {
 		if item.expectUpdate {
 			actualEvent := <-patchEvent
 			t.Logf("%v - validating event affected by patch request", index)
-			validateEventSerie(strconv.Itoa(index), true, actualEvent, item.expect, t)
+			validateEventSeries(strconv.Itoa(index), true, actualEvent, item.expect, t)
```

Comment:
actually, we should just rename this to `validateEvent`

Reply:
done